products/health-care/appointments/va-online-scheduling/research/may-2021-facilities-personalization-research/conversation-guide.md

```diff
@@ -82,9 +82,9 @@ So where do you currently receive health care, VA or otherwise?
 
 -   How many different VA locations have you gone to for care?
 
-    -   If more than one: why do you go to some of the other locations?
+    -   If more than one: why do you go to some of the other locations? ```comment: This question works well if they are going to multiple facilities currently, but if they say that they have gone to more than one location b/c they moved or something in the question before this, this question doesn't make sense. However, it might be worth asking them directly in a separate question if they currently go to more than one VA facility for care. Be preapred - You may get answers like CBOCs and main facilities, which isn't quite what you're looking for. ```
 
-    -   Does anything stop you from going to a different location that might be better suited to your needs?
+    -   Does anything stop you from going to a different location that might be better suited to your needs? ```comment: Are you looking for VA specific here, or all health care? May need to specify.``` 
```

Comment:
This is broad and more focused toward preferences, so actually it's covered by "If no would you like other options" above. I'll cut it.